autoload/fzf/vim.vim

```diff
@@ -687,18 +687,30 @@ function! s:bufopen(lines)
   if len(a:lines) < 2
     return
   endif
-  let b = matchstr(a:lines[1], '\[\zs[0-9]*\ze\]')
-  if empty(a:lines[0]) && get(g:, 'fzf_buffers_jump')
+
+  let key = a:lines[0]
+  let buffer_numbers = map(a:lines[1:], {_idx, bufline -> matchstr(bufline, '\[\zs[0-9]*\ze\]')})
+
+  if key == 'ctrl-d'
+    execute 'bdelete' join(buffer_numbers, ' ')
+    return
+  endif
+
+  let b = buffer_numbers[0]
```

Comment:
When multiple buffers are selected we open only the first one, not sure if this is the right/desired behaviour

Reply:
So we can delete multiple buffer at once? just like we open multiple files using tab in `:Files` or there's a different mechanism here?